splice_machine/src/test/java/com/splicemachine/derby/impl/sql/execute/operations/VTIOperationIT.java

```diff
@@ -180,18 +181,91 @@ public void testFileVTIExpectError() throws Exception {
     @Test
     public void testFileVTITypes() throws Exception {
         String location = getResourceDirectory()+"vtiConversion.in";
-        String sql = String.format("select * from new com.splicemachine.derby.vti.SpliceFileVTI('%s','',',') as b" +
+        String sql = String.format("select '-' || name || '-', '-' || title || '-', age, something, date_hired, clock from new com.splicemachine.derby.vti.SpliceFileVTI('%s','',',') as b" +
                                        " (name varchar(10), title varchar(30), age int, something varchar(12), " +
-                                       "date_hired timestamp, clock time)\n" +
+                                       "date_hired timestamp, clock time) --splice-properties useSpark=false\n" +
                                        " where age < 40 and date_hired > TIMESTAMP('2015-08-21', '08:09:08') order" +
                                        " by name", location);
         ResultSet rs = spliceClassWatcher.executeQuery(sql);
         String expected =
-            "NAME  |          TITLE          | AGE | SOMETHING  |     DATE_HIRED       |  CLOCK  |\n" +
-                "--------------------------------------------------------------------------------------\n" +
-                "jzhang | How The West Won Texas  | 34  |08-23X-2015 |2015-08-22 08:12:08.0 |11:08:08 |\n" +
-                "sfines |Senior Software Engineer | 27  |08X-27-2015 |2015-08-27 08:08:08.0 |06:08:08 |";
+            "1    |             2             | AGE | SOMETHING  |     DATE_HIRED       |  CLOCK  |\n" +
+                    "------------------------------------------------------------------------------------------\n" +
+                    "-jzhang- | -How The West Won Texas-  | 34  |08-23X-2015 |2015-08-22 08:12:08.0 |11:08:08 |\n" +
+                    "-sfines- |-Senior Software Engineer- | 27  |08X-27-2015 |2015-08-27 08:08:08.0 |06:08:08 |";
+        assertEquals("\n"+sql+"\n", expected, TestUtils.FormattedResult.ResultFactory.toStringUnsorted(rs));
+        rs.close();
+
+        //test spark path
+        sql = String.format("select '-' || name || '-', '-' || title || '-', age, something, date_hired, clock from new com.splicemachine.derby.vti.SpliceFileVTI('%s','',',') as b" +
+                " (name varchar(10), title varchar(30), age int, something varchar(12), " +
+                "date_hired timestamp, clock time) --splice-properties useSpark=false\n" +
+                " where age < 40 and date_hired > TIMESTAMP('2015-08-21', '08:09:08') order" +
```

Comment:
Comment says "test spark path" but it sets "useSpark=false".

Reply:
Thanks. I'll fix it.